src/rules/colon_assignment_spacing.coffee

```diff
@@ -57,7 +60,8 @@ module.exports = class ColonAssignmentSpacing
         [isLeftSpaced, leftSpacing] = checkSpacing 'left'
         [isRightSpaced, rightSpacing] = checkSpacing 'right'
 
-        if isLeftSpaced and isRightSpaced
+        if token.csxColon or isLeftSpaced and isRightSpaced
```

Comment:
we have no control over spacing of "csxColons" (these are autogenerated by the compiler)